### PR TITLE
feat: Set up basic commands that are called from cmd/main.go

### PR DESCRIPTION
**Pull request by whutchinson98**
Commands are:
`help` -> provides list of commands with descriptions
`version` -> provides go version, goos and goarch

### CODE REVIEW

File: pkg/commands/version.go [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package commands
+
+import (
+	"fmt"
+	"runtime"
+)
+
+func VersionCommand(commands *[]string, flags *map[string]interface{}) error {
```
Comment (jmaister) [inline pkg/commands/version.go:8]: Each Command class could be implement a BaseCommand struct like this:

```go
type Command struct {
	Name        string
	Description string
    Run     func(commands *[]string, flags *map[string]interface{})
}
```

Comment (whutchinson98) [reply pkg/commands/version.go:8]: Great idea! I'll implement that tonight thank you

Comment (whutchinson98) [reply pkg/commands/version.go]: Updated now, got excited :D

**Review by jmaister — APPROVED** [review]

**Review by jmaister — APPROVED** [review]
Looks great!

**Pull request merged**